### PR TITLE
feat(fac): Expose FAC APIs from firebase-admin/app-check entry point

### PR DESCRIPTION
**Pull request by lahirumaramba**
- Migrated FAC APIs to modular SDK
- Exposed FAC API types from the firebase-admin/app-check entry point.
- Added API report for new entry point.

### CODE REVIEW

File: src/app-check/app-check-api.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+  /**
+   * The time-to-live duration of the token in milliseconds.
+   */
+  ttlMillis: number;
+}
+
+/**
+ * Interface representing a decoded Firebase App Check token, returned from the
+ * {@link appCheck.AppCheck.verifyToken `verifyToken()`} method.
```
Comment (hiranya911) [inline src/app-check/app-check-api.ts:35]: This needs to follow the conventions described at https://tsdoc.org/pages/tags/link/

File: src/app-check/app-check-api.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+ */
+export interface DecodedAppCheckToken {
+  /**
+   * The issuer identifier for the issuer of the response.
+   *
+   * This value is a URL with the format
+   * `https://firebaseappcheck.googleapis.com/<PROJECT_NUMBER>`, where `<PROJECT_NUMBER>` is the
+   * same project number specified in the [`aud`](#aud) property.
```
Comment (hiranya911) [inline src/app-check/app-check-api.ts:43]: Same here. Use the correct tsdoc syntax to refer to aud property.

You can try running `npm run apidocs` and inspect the markdown output to see what it looks like.

**Review by hiranya911 — CHANGES_REQUESTED** [review]
Thanks. This looks great! Only had a couple of minor comments regarding the references in API ref docs.

File: src/app-check/app-check-api.ts (hunk 2 above)
Comment (lahirumaramba) [reply src/app-check/app-check-api.ts]: Thanks! Good catch. Used @link to fix these.

File: src/app-check/app-check-api.ts (hunk 1 above)
Comment (lahirumaramba) [reply src/app-check/app-check-api.ts]: Thanks! Fixed the links.

**Review by hiranya911 — APPROVED** [review]
LGTM 👍

**Pull request merged**